src/tasks/MonoTargetsTasks/ILStrip/ILStrip.cs

```diff
@@ -143,7 +171,21 @@ private bool TrimMethods(ITaskItem assemblyItem)
             return true;
         }
 
-        string trimmedAssemblyFilePath = ComputeTrimmedAssemblyPath(assemblyFilePath);
+        string trimmedAssemblyFilePath = ComputeTrimmedAssemblyPath(trimmedAssemblyFolder, assemblyFilePath);
+        if (File.Exists(trimmedAssemblyFilePath))
+        {
+            if (IsInputNewerThanOutput(assemblyFilePath, trimmedAssemblyFilePath))
```

Comment:
Please log whether the file is being trimmed or being skipped, and why. See https://github.com/dotnet/runtime/blob/25961d1156b3ed8cf684aaa9c56d78a8b2cecdfe/src/tasks/WasmAppBuilder/EmccCompile.cs#L251-L296 for an example.

Reply:
Added.